humanmine.ipynb

```diff
@@ -0,0 +1,572 @@
+    "    qs3 = {'key': TYPE, 'value': gene, 'project_id': project_id}\n",
+    "    url3 = DATASETS_URL.format(**qs3)\n",
+    "    find_images(session.get(url3).json(), \"datasets\")\n",
+    "\n",
+    "print(images)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 92,
+   "id": "sunset-playlist",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BASE_URL = \"https://idr.openmicroscopy.org/webclient\"\n",
+    "IMAGE_DATA_URL = BASE_URL + \"/render_thumbnail/{id}\"\n",
+    "LINK_URL = BASE_URL + \"/?show=image-{id}\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "subjective-radius",
+   "metadata": {},
+   "source": [
+    "## Display the images"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 120,
+   "id": "detailed-oakland",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<table><tr><td><h4>Development stage: 15PCW</h4></td></tr><tr><td><a href=\"https://idr.openmicroscopy.org/webclient/?show=image-9841210\" target=\"_blank\"><img src=\"https://idr.openmicroscopy.org/webclient/render_thumbnail/9841210\"/></a></td><td><a href=\"https://idr.openmicroscopy.org/webclient/?show=image-9841211\" target=\"_blank\"><img src=\"https://idr.openmicroscopy.org/webclient/render_thumbnail/9841211\"/></a></td></tr><tr><td><h4>Development stage: 9PCW</h4></td></tr><tr><td><a href=\"https://idr.openmicroscopy.org/webclient/?show=image-9841212\" target=\"_blank\"><img src=\"https://idr.openmicroscopy.org/webclient/render_thumbnail/9841212\"/></a></td><td><a href=\"https://idr.openmicroscopy.org/webclient/?show=image-9841219\" target=\"_blank\"><img src=\"https://idr.openmicroscopy.org/webclient/render_thumbnail/9841219\"/></a></td><td><a href=\"https://idr.openmicroscopy.org/webclient/?show=image-9841213\" target=\"_blank\"><img src=\"https://idr.openmicroscopy.org/webclient/render_thumbnail/9841213\"/></a></td><td><a href=\"https://idr.openmicroscopy.org/webclient/?show=image-9841218\" target=\"_blank\"><img src=\"https://idr.openmicroscopy.org/webclient/render_thumbnail/9841218\"/></a></td></tr><tr><td><h4>Development stage: CS16</h4></td></tr><tr><td><a href=\"https://idr.openmicroscopy.org/webclient/?show=image-9841216\" target=\"_blank\"><img src=\"https://idr.openmicroscopy.org/webclient/render_thumbnail/9841216\"/></a></td><td><a href=\"https://idr.openmicroscopy.org/webclient/?show=image-9841217\" target=\"_blank\"><img src=\"https://idr.openmicroscopy.org/webclient/render_thumbnail/9841217\"/></a></td></tr><tr><td><h4>Development stage: CS21</h4></td></tr><tr><td><a href=\"https://idr.openmicroscopy.org/webclient/?show=image-9841215\" target=\"_blank\"><img src=\"https://idr.openmicroscopy.org/webclient/render_thumbnail/9841215\"/></a></td><td><a href=\"https://idr.openmicroscopy.org/webclient/?show=image-9841214\" target=\"_blank\"><img src=\"https://idr.openmicroscopy.org/webclient/render_thumbnail/9841214\"/></a></td></tr></table>"
```

Comment:
```suggestion
       "<table><tr><td><h4>Development stage: 15PCW</h4></td></tr><tr><td><a href=\"https://idr.openmicroscopy.org/webclient/?show=image-9841210\" target=\"_blank\"><img src=\"https://idr.openmicroscopy.org/webclient/render_thumbnail/9841210\"/></a></td><td><a href=\"https://idr.openmicroscopy.org/webclient/?show=image-9841211\" target=\"_blank\"><img src=\"https://idr.openmicroscopy.org/webclient/render_thumbnail/9841211\"/></a></td></tr><tr><td><h4>Developmental stage: 9PCW</h4></td></tr><tr><td><a href=\"https://idr.openmicroscopy.org/webclient/?show=image-9841212\" target=\"_blank\"><img src=\"https://idr.openmicroscopy.org/webclient/render_thumbnail/9841212\"/></a></td><td><a href=\"https://idr.openmicroscopy.org/webclient/?show=image-9841219\" target=\"_blank\"><img src=\"https://idr.openmicroscopy.org/webclient/render_thumbnail/9841219\"/></a></td><td><a href=\"https://idr.openmicroscopy.org/webclient/?show=image-9841213\" target=\"_blank\"><img src=\"https://idr.openmicroscopy.org/webclient/render_thumbnail/9841213\"/></a></td><td><a href=\"https://idr.openmicroscopy.org/webclient/?show=image-9841218\" target=\"_blank\"><img src=\"https://idr.openmicroscopy.org/webclient/render_thumbnail/9841218\"/></a></td></tr><tr><td><h4>Developmental stage: CS16</h4></td></tr><tr><td><a href=\"https://idr.openmicroscopy.org/webclient/?show=image-9841216\" target=\"_blank\"><img src=\"https://idr.openmicroscopy.org/webclient/render_thumbnail/9841216\"/></a></td><td><a href=\"https://idr.openmicroscopy.org/webclient/?show=image-9841217\" target=\"_blank\"><img src=\"https://idr.openmicroscopy.org/webclient/render_thumbnail/9841217\"/></a></td></tr><tr><td><h4>Developmental stage: CS21</h4></td></tr><tr><td><a href=\"https://idr.openmicroscopy.org/webclient/?show=image-9841215\" target=\"_blank\"><img src=\"https://idr.openmicroscopy.org/webclient/render_thumbnail/9841215\"/></a></td><td><a href=\"https://idr.openmicroscopy.org/webclient/?show=image-9841214\" target=\"_blank\"><img src=\"https://idr.openmicroscopy.org/webclient/render_thumbnail/9841214\"/></a></td></tr></table>"
```

Reply:
3 replacements of Development -> Developmental

Reply:
Ignore that.